dev/DeploymentAgent/tracelogging.h

```diff
@@ -15,10 +15,11 @@ class WindowsAppRuntimeDeploymentAgent_TraceLogger final : public wil::TraceLogg
 
 public:
 
-    DEFINE_COMPLIANT_CRITICAL_DATA_EVENT_PARAM3(
+    DEFINE_COMPLIANT_CRITICAL_DATA_EVENT_PARAM4(
         Success,
         PDT_ProductAndServicePerformance,
-        PCWSTR, packagePath,
+        bool, isRegisterHigherVersion,
```

Comment:
You're changing the data/order in the event. That's not a problem?

Reply:
Order is not important. That's no problem.